tiledb/sm/cpp_api/stats.h

```diff
@@ -35,6 +35,8 @@
 
 #include "tiledb.h"
 
+#include <string>
+
```

Comment:
Why was this included?

Reply:
Must have been from a transient state where a source file was including fewer things. I'm not convinced that these headers actually include everything they need.

In any case things appear to build fine without this now, so I've reverted it.

Reply:
Oh I didn't see the `std::string` later in the file. You can put the header back if you want I don't mind either way.